src/v/cluster/cloud_metadata/key_utils.h

```diff
@@ -43,4 +43,8 @@ cloud_storage_clients::object_key offsets_snapshot_key(
   const model::partition_id&,
   size_t snapshot_idx);
 
+// E.g. /wcr_cluster_name/<name>/uuid/<uuid>
```

Comment:
I mentioned a similar point in the doc but I think it's worth thinking about whether we want this to actually be tightly coupled to "WCR" at the storage layer. Maybe we should just call the path prefix `cluster_name`?

At least with cluster properties we can alias